models/zero_point.py

```diff
@@ -17,9 +18,19 @@ class ZeroPoint(Base, AutoIDMixin):
 
     source_list = orm.relationship(
         'SourceList',
+        lazy='selectin',
         doc="The source list this zero point is associated with. "
     )
 
+    # TODO : figure this out
+    # image = orm.relationship(
+    #     'Image',
+    #     secondary='source_lists',
+    #     primaryjoin='zero_points.c.source_list_id == source_lists.c.id',
+    #     secondaryjoin='source_lists.c.image_id == images.c.id',
+    #     single_parent=True
+    # )
```

Comment:
I think you can do this really easily with an `associationproxy`. 

https://docs.sqlalchemy.org/en/20/orm/extensions/associationproxy.html#scalar-relationships